data/files/table1

```diff
@@ -0,0 +1,5 @@
+1	Acura	4
```

Comment:
can we give the file a non generic name?

Reply:
sure, done